CorsixTH/Lua/map.lua

```diff
@@ -180,8 +180,9 @@ function Map:load(level, difficulty, level_name, map_file, level_intro, map_edit
   local result = file(path .. "Lua" .. pathsep .. "base_config.lua")
 
   local base_config = result
-  local errors
+  local _
```

Comment:
I see, luacheck allows shadowing _, but not referencing it as a global. Fine then, we'll follow their lead.